templates/create_movies_form.html

```diff
@@ -1,9 +1,106 @@
+<!-- Links below used for help creating star rating system: -->
+<!-- https://gist.github.com/prof3ssorSt3v3/29e623d441e8174ffaef17741a1bba14 -->
+
 {% extends '_layout.html' %}
 
 {% block title %}Movie Rating App | Create Movie Rating{% endblock %}
 
 {% block body %}
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Bootstrap is already included in `_layout.html` and the fontawesome library (great library btw, we use at my job) should also be included in `_layout.html` because that's where your `head` tag is.